modules/redshift-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/redshift/Redshift.scala

```diff
@@ -170,13 +222,16 @@ object Redshift {
                      | ACCEPTINVCHARS
                      | $frCompression""".stripMargin
 
-              case Statement.ShreddedCopy(shreddedType, compression, loadAuthMethod) =>
-                val frTableName = Fragment.const0(qualify(shreddedType.info.getName))
+              case Statement.ShreddedCopy(shreddedType, compression, loadAuthMethod, shredModel, tableName) =>
+                val frTableName = Fragment.const0(qualify(tableName))
                 val frPath = Fragment.const0(shreddedType.getLoadPath)
                 val frCredentials = loadAuthMethodFragment(loadAuthMethod, storage.roleArn)
                 val frRegion = Fragment.const0(region.name)
                 val frMaxError = Fragment.const0(maxError.toString)
                 val frCompression = getCompressionFormat(compression)
+                val extraCols = ShredModelEntry.extraCols.map(_._1.replaceAll(""""""", ""))
```

Comment:
I guess you have local changes to schema-ddl which you have not pushed to github yet?  I think I need to see those changes before I comment on this section.

Reply:
you're right, I just pushed a new commit at https://github.com/snowplow/schema-ddl/pull/193